ckanext/datawagovautheme/templates/snippets/tag_list.html

```diff
@@ -0,0 +1,10 @@
+{% ckan_extends %}
```

Comment:
What do these additions do?

Reply:
this bit of code is overriding the ellipsis for the tags get display below search bar on dataset page.
https://uat.landgate.links.com.au/dataset

Reply:
I've clean up commented code.